builtin/credential/ldap/path_config.go

```diff
@@ -143,8 +166,22 @@ func (b *backend) pathConfigWrite(
 	if groupdn != "" {
 		cfg.GroupDN = groupdn
 	}
+	groupfilter := d.Get("groupfilter").(string)
```

Comment:
Negative, all search scenarios now depend on `groupfilter`. If the user doesn't specify it, they need to get the default which will give them the legacy heuristic behavior.